src/cli/index.ts

```diff
@@ -81,3 +83,20 @@ main().catch((e) => {
   console.error(e.stack);
   process.exit(1);
 });
+
+/** Run up project tree to find .projen directory
+ *
+ * @param cwd current working directory
+ * @returns path to .projen directory or undefined if not found
+ */
+function findProjenDir(cwd: string): string | undefined {
+  if (cwd === "/") {
+    return undefined;
+  }
+
+  if (fs.existsSync(resolve(cwd, PROJEN_DIR))) {
+    return cwd;
+  }
```

Comment:
I think technically these need to be switched around if you want to allow `/.projen`
```suggestion
  if (fs.existsSync(resolve(cwd, PROJEN_DIR))) {
    return cwd;
  }

  if (cwd === "/") {
    return undefined;
  }
```